app/views/request/_sidebar.html.erb

```diff
@@ -2,17 +2,6 @@
   <div class="new-request-cta">
     <%= render :partial => "general/new_request" %>
   </div>
-  <div id="track-request" class="track-request">
-    <h2><%= _('Follow this request') %></h2>
-
-    <p>
-      <%= n_("There is {{count}} person following this request",
-             "There are {{count}} people following this request",
-             @follower_count,
-             :count => @follower_count) %>
-    </p>
```

Comment:
Just for reference, here's what we do for people following authorities:

![screen shot 2016-09-12 at 11 24 37](https://cloud.githubusercontent.com/assets/282788/18432732/9aa62d64-78db-11e6-978c-59d768ea07fd.png)